FSJ_django20_project/FSJ/forms_adjudicator.py

```diff
@@ -0,0 +1,24 @@
+from .models_adjudicator import Adjudicator
+from django.forms import ModelForm
+
+class AdjudicatorForm(ModelForm):
+
+    class Meta:
+        model = Adjudicator
+        exclude = ()
```

Comment:
Should we not be explicitly adding accepted fields versus implicitly? As we add fields to various fsj user objects I think we'd want to intentionally include them on add/edit forms as some fields such as language preference aren't really up to to the coordinator to select for a student.

Reply:
Not sure which fields a coordinator can or cannot edit, felt safer to leave them all in for the time being. Easy fix if I know exactly what to include/exclude.

Reply:
Perhaps we can explicitly add every field at present then? That way, they are all safely there and future fields getting added to models, we would need to actively choose their inclusion instead of having it just happen without our knowledge?

Reply:
Will do

Reply:
Now fixed